src/platform/riot/coap_udp_ip_channel.c

```diff
@@ -7,17 +7,15 @@
 #include "net/sock/util.h"
 #include <arpa/inet.h>
 
-static bool _is_coap_initialized = false;
+static bool _is_globals_initialized = false;
```

Comment:
we need those globals for some kind of handler function?

Reply:
Yes I need `Environment` to get the channel bundle list in most callbacks to find the channel based on its socket. (See get_channel_by_remote or similar?)